pkg/asset/handler/get.go

```diff
@@ -114,6 +115,13 @@ func (h *GetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return nil
 		}
 
+		// Remove existing CORS header in response
```

Comment:
Leave a TODO here to support custom Access-Control-* headers

Reply:
I think we already support them? The proprietary headers mapping contains the CORS header.

Reply:
I meant your TODO in this PR description. To support custom CORS header in gateway.

Reply:
I think I'll open an issue instead, since I'm not sure where should it be implemented.

Reply:
ref #1135